packages/sdk/src/types.ts

```diff
@@ -38,23 +38,32 @@ export interface MakeOrderParams {
 
   /** Expiration time for the order */
   expirationTime?: Date;
+
+  /** Selected marketplaces */
+  marketplaces?: Array<`${MarketplaceName}`>;
```

Comment:
String literal trick came from here https://stackoverflow.com/a/56843823/2743158
Without it there was no way to use enum as value here.

Reply:
Should we stick to array literal? `${MarketplaceName}`[]